src/components/TextArea/index.js

```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import "./TextArea.css";
+
+function TextArea({ name, placeholder }) {
+  return (
+    <textarea
+      name={name}
+      placeholder={placeholder}
+      rows="5"
```

Comment:
@AbdurrahmanSogbesan  let rows be a prop. You also have to add an onChange prop if not you wont be able to use the textbox for functional things